samples/sqltransport/multi-instance/SqlServer_3/Sender/Program.cs

```diff
@@ -68,4 +69,25 @@ await endpoint.Send("Samples.SqlServer.MultiInstanceReceiver", order)
         Console.WriteLine($"ClientOrder message sent with ID {order.OrderId}");
     }
 
+    #region SenderConnectionProvider
+    static class ConnectionProvider
+    {
+        const string DefaultConnectionString = @"Data Source=.\SqlExpress;Database=SenderCatalog;Integrated Security=True";
+        const string ReceiverConnectionString = @"Data Source=.\SqlExpress;Database=ReceiverCatalog;Integrated Security=True";
+
+        public static async Task<SqlConnection> GetConnection(string transportAddress)
+        {
+            var connectionString = transportAddress.StartsWith("Samples.SqlServer.MultiInstanceReceiver")
```

Comment:
I don't understand why the check is inverse? i.e. the project is Sender, but it checks the transport address to that of the Receiver. 
Shouldn't the check be  `Samples.SqlServer.MultiInstanceSender` instead of `Samples.SqlServer.MultiInstanceReceiver` as this is in the `Sender/Program.cs` ?

Wouldn't the transport address be MultiInstanceReceiver in this project?